common/src/main/java/com/wynntils/models/statuseffects/StatusEffectModel.java

```diff
@@ -87,7 +90,18 @@ public void onTabListCustomization(PlayerInfoFooterChangedEvent event) {
             StyledText prefix = parts.get(0);
             StyledText name = parts.get(1);
             StyledText displayedTime = parts.get(2);
-            newStatusEffects.add(new StatusEffect(name, displayedTime, prefix));
+            StyledText modifier;
+
+            // Split the modifier and name, which are separated by a space
+            StyledText[] modifierStr = name.split("\\s{1}");
+            if (modifierStr[0].matches(MODIFIER_REGEX)) {
+                modifier = modifierStr[0];
+                name = name.substring(modifier.length()).trim(); // Get all but first part of the string
+            } else {
+                modifier = StyledText.EMPTY;
+            }
```

Comment:
I would like to see you create a new group in the original STATUS_EFFECT_PATTERN and just get it, without doing this whole modification. 

Is there a reason why you didn't do that? I can help you with it if needed.

Reply:
I have given it another try and can confirm that `.group()` messes up the styles. The `name` part and the `modifier` part share the same token for giving the style, which would have to be included in the capture group in the regex.

Reply:
Okay, then, let's keep this open for now and I will think about this.